pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "dialog-lib"
-version = "0.0.1.19"
+version = "0.0.1.20"
```

Comment:
wouldn't this be a minor increment, adding a new functionality?

Reply:
@lgabs good question... Honestly, I don't know. We could bump it for sure